Python/peephole.c

```diff
@@ -474,10 +475,14 @@ PyCode_Optimize(PyObject *code, PyObject* consts, PyObject *names,
         unsigned int offset_delta, new_offset;
         cum_orig_offset += lnotab[i];
         assert(cum_orig_offset % sizeof(_Py_CODEUNIT) == 0);
-        new_offset = blocks[cum_orig_offset / sizeof(_Py_CODEUNIT)] *
-                sizeof(_Py_CODEUNIT);
-        offset_delta = new_offset - last_offset;
-        assert(offset_delta <= 255);
+        index = cum_orig_offset / sizeof(_Py_CODEUNIT);
```

Comment:
Why is this change necessary?

Reply:
Because the current line processing produces temporary indexes that violate that the `offset_delta < 255` when you run the optimizer multiple times. You can comment this change and experiment for yourself. This is the smaller and less invasive change to the peephole optimizer itself that I can think of, but I welcome any idea around this,

Reply:
Again, can you add a brief comment to the code. thanks

Reply:
Will do :)